homeassistant/components/yale_smart_alarm/strings.json

```diff
@@ -0,0 +1,22 @@
+{
+    "config": {
+      "abort": {
+        "already_configured": "[%key:common::config_flow::abort::already_configured_account%]"
+      },
+      "error": {
+        "auth_error": "[%key:common::config_flow::error::invalid_auth%]",
+        "connection_error": "[%key:common::config_flow::error::cannot_connect%]"
+      },
+      "step": {
+        "user": {
+          "data": {
+            "username": "[%key:common::config_flow::data::username%]",
+            "password": "[%key:common::config_flow::data::password%]",
+            "code": "Code to open/close door lock",
+            "name": "Name of integration",
```

Comment:
Shouldn't name say "Yale Smart Living"?

Reply:
It should say "Name of Alarm" as it would be the name of the alarm system (as none is supplied by the system). Want to give the user an option. Also changed now in new commit